Backend/AMQPBackend.php

```diff
@@ -30,6 +30,9 @@ class AMQPBackend implements BackendInterface
 
     protected $queue;
 
+    /**
+     * @deprecated This property is deprecated since version 2.3 and will be removed in 3.0.
```

Comment:
Since 2.4.

2.3 is already stable.

BTW, a new branch should be created before merging this PR.